.github/workflows/pr-build.yml

```diff
@@ -37,9 +66,41 @@ jobs:
       - name: Build with Flutter
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: flutter build apk --debug
+        run: |
+          flutter build apk --${{ inputs.app-variant }};
+      - name: Set env
+        run: |
+          echo "COMMIT_HASH=$(git rev-parse --short HEAD)" >> $GITHUB_ENV
       - name: Upload build
         uses: actions/upload-artifact@v3
         with:
-          name: revanced-manager
-          path: build/app/outputs/flutter-apk/app-debug.apk
+          name: revanced-manager-${{ env.COMMIT_HASH }}-${{ inputs.pr-number }}-${{ inputs.app-variant }}-${{ inputs.flutter-variant }}
+          path: build/app/outputs/flutter-apk/app-${{ inputs.app-variant }}.apk
+          
+  notify-pull-request:
+    name: Notify Pull Request
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - name: Set env
+        run: |
+          echo "DATETIME=$(date +'%Y-%m-%d %H:%M:%S')" >> $GITHUB_ENV
+          if [[ "${{ needs.build.result }}" == "failure" ]]; then
+            echo "MESSAGE=🚫 Build failed!" >> $GITHUB_ENV
+          else
+            echo "MESSAGE=✅ Build succeeded!" >> $GITHUB_ENV
+          fi
```

Comment:
TODO: this is always true (or false, maybe)

Reply:
Has this been resolved?

Reply:
> Has this been resolved?

I pushed a (unverified) fix, that stop the CI when error has occurred.

Reply:
Can you check if the workflow pushed an asset?

Reply:
> Can you check if the workflow pushed an asset?

Artifact? Yep.

* APK
* Sha1

Reply:
Fixed in latest revision

![image](https://github.com/ReVanced/revanced-manager/assets/93124920/96d72529-ddc0-48de-a7fe-89c2d5678eef)